src/components/Reports/Reports.test.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { render, cleanup, wait } from 'react-testing-library';
+import 'jest-dom/extend-expect';
+import DopplerIntlProvider from '../../DopplerIntlProvider.double-with-ids-as-values';
+
+import { getDomains, getPagesByDomainId } from './ReportsService';
+import Reports from './Reports';
+
+jest.mock('./ReportsService');
+
+const fakeData = [
+  {
+    id: 1,
+    name: 'www.fromdoppler.com',
+    verified_date: '17/12/2017',
+  },
+  {
+    id: 2,
+    name: 'www.makingsense.com',
+    verified_date: '17/12/2020',
+  },
+];
+
+const fakePages = [{ id: '1', name: 'productos2' }, { id: '2', name: 'servicios2' }];
+
+describe('Reports page', () => {
+  afterEach(cleanup);
+
+  beforeEach(() => {
+    getDomains.mockClear();
+    getPagesByDomainId.mockClear();
+  });
+
+  it('load domains without pages', () => {
```

Comment:
Is this test redundant? 

Could you rephrase it as an assertion sentence, like _"it (Reports page) should render domains without pages"_

Reply:
fixed